addon-test-support/ember-qunit/index.js

```diff
@@ -228,6 +233,18 @@ export function setupEmberOnerrorValidation() {
   });
 }
 
+export function setupAsyncTimerLeakDetection() {
+  QUnit.testDone(({ module, name }) => {
```

Comment:
Can we make this whole function the thing that is extracted?  So that the resulting code would be:

```js
QUnit.testDone(importedThingGoesHere);
QUnit.done(otherImportedThingGoesHere);
```